gulpfile.js

```diff
@@ -68,6 +68,12 @@ gulp.task('fonts-libs', function() {
         .pipe(gulp.dest(path.join(output_directory, 'fonts')));
 });
 
+// Catches img file for angular-chosen.
```

Comment:
Is there more than one file? If yes, type `// Catches image files for angular-chosen.`